kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/controller/DefaultRestControllerImpl.java

```diff
@@ -285,4 +286,18 @@ public void stopContainer(String containerId) {
             }
         }
     }
+
+    private String loadPassword(KieServerConfig config) {
```

Comment:
This method seems like exact copy of the one in WebSocket.*Client. Please move the method to some utility class to avoid duplication.

Reply:
@rsynek fixed